ntp_test.go

```diff
@@ -1,7 +1,10 @@
 package ntp
 
 import (
+	"github.com/stretchr/testify/assert"
```

Comment:
Any chance you could write the tests without pulling in a third-party dependency?  I'd prefer to use only go's standard packages if possible.